docs/examples/floating-pv/plot_floating_pv_cell_temperature.py

```diff
@@ -0,0 +1,241 @@
+# systems.
+
+# Make a dictionary containing all the sets of coefficients presented in the
+# above table.
+heat_loss_coeffs = {
+    'open_structure_small_footprint_tracking_NL': [24.4, 6.5, 'C0', 'solid'],
+    'open_structure_small_footprint_tracking_NL_2': [57, 0, 'C0', 'dashed'],
+    'closed_structure_large_footprint_NL': [25.2, 3.7, 'C1', 'solid'],
+    'closed_structure_large_footprint_NL_2': [37, 0, 'C1', 'dashed'],
+    'closed_structure_large_footprint_SG': [34.8, 0.8, 'C2', 'solid'],
+    'closed_structure_large_footprint_SG_2': [36, 0, 'C2', 'dashed'],
+    'closed_structure_medium_footprint_SG': [18.9, 8.9, 'C3', 'solid'],
+    'closed_structure_medium_footprint_SG_2': [41, 0, 'C3', 'dashed'],
+    'open_structure_free_standing_SG': [35.3, 8.9, 'C4', 'solid'],
+    'open_structure_free_standing_SG_2': [55, 0, 'C4', 'dashed'],
+    'in_contact_with_water_NO': [71, 0, 'C5', 'solid'],
+    'open_strucutre_free_standing_IT': [31.9, 1.5, 'C6', 'solid'],
+    'open_strucutre_free_standing_bifacial_IT': [35.2, 1.5, 'C7', 'solid'],
+    'default_PVSyst_coeffs_for_land_systems': [29.0, 0, 'C8', 'solid']
+}
+
+# Plot the cell temperature for each set of the above heat loss coefficients
+for coeffs in heat_loss_coeffs:
+    T_cell = pvlib.temperature.pvsyst_cell(
+        poa_global=irradiance['poa_global'],
+        temp_air=tmy['temp_air'],
+        wind_speed=tmy['wind_speed'],
+        u_c=heat_loss_coeffs[coeffs][0],
+        u_v=heat_loss_coeffs[coeffs][1]
+    )
+    # Convert Dataframe Indexes to Hour format to make plotting easier
+    T_cell.index = T_cell.index.strftime("%H")
+    plt.plot(T_cell, label=coeffs, c=heat_loss_coeffs[coeffs][2],
+             ls=heat_loss_coeffs[coeffs][3], alpha=0.8)
+
+plt.xlabel('Hour')
+plt.ylabel('PV cell temperature\n$[°C]$')
+plt.ylim(20, 45)
+plt.xlim('06', '20')
+plt.grid()
+plt.legend(loc='upper left', frameon=False, ncols=2, fontsize='x-small',
+           bbox_to_anchor=(0, -0.2))
+plt.tight_layout()
+plt.show()
+
+# %%
+# The figure above illustrates the necessity of choosing appropriate heat loss
+# coefficients when using the PVSyst model for calculating the cell temperature
+# for floating PV systems. A difference of up to 10.3 °C was obtained when
+# using the default PVSyst coefficients and the coefficients when the panels
+# are in contact with water.
+#
+# It should be noted that, for the systems having both a single U-value and
+# a combination of Uc and Uv, approximately the same results were obtained
+# in the literature. However, in this example, there is a difference in the
+# calculated cell temperatures. The reason is that the wind speed in the
+# presented example is probably quite different than the one measured
+# in the corresponding test sites, making the division between
+# wind-dependent and -independent heat loss coefficients less accurate for wide
+# use. Thus, it is suggested to use the single heat loss U-values for system
+# simulations.
```

Comment:
To me, it seems like this makes the wrong conclusion.  A single U value would have a particular wind speed "baked into" it, meaning it would not generalize well to other climates.  Separate Uc, Uv parameters is what enables more accurate use for other wind speeds.  If we make a recommendation, shouldn't it be to use Uc and Uv instead of U?  Maybe I misunderstand. 

However, I am not sure we should make a recommendation at all.  My two cents: I think pointing out the discrepancy and speculating on the source is appropriate, but I think suggested solutions should be backed up by a reference.

Reply:
My last sentence was based on the comment from @lmicheli above:

> I think, as for LPV, the single U-value is most commonly employed. So I would suggesting adding the single U-values as well (or replacing the Uc/Uv with the single values).

But I definitely see your point @kandersolar! I will remove the last sentence.